### DISCUSSION TITLE
Cant login when running locally

### DISCUSSION THREAD

**Discussion by sama8525** [reworkd/AgentGPT#851] [Q&A] [answered]
I think i have everything set up right, had to do it manually as docker refuses to run without some sort of bios setting.

Both the front and back end seem to operate, i can navigate to the page in the browser, but it wont let me do anything unless i sign in, and everything i try for signing in errors out. What am i doing wrong? is theres something i need to do to get the database to work?

thanks,
![Screenshot 2023-06-23 011403](https://github.com/reworkd/AgentGPT/assets/124485155/60c6d55b-0782-4f8c-bad1-b390e7989e82)
![Screenshot 2023-06-23 011340](https://github.com/reworkd/AgentGPT/assets/124485155/2fefff55-2960-453d-b518-d15425118819)

**Comment by jasangill1** [comment, marked as answer]
Hello @sama8525 if you are not running through docker you must have a database instance setup somewhere. Typically this is on 3306 (But on docker this is 3307). You can update your ENV file to change the DB_PORT for both the frontend and backend (platform) to which ever port your running instances are using. 

Alternatively, you can try to fix your docker settings to avoid this.
